container/worker/run_openqa_worker.sh

```diff
@@ -24,4 +24,14 @@ fi
 qemu-system-x86_64 -S &
 kill $!
 
+if [[ -e /var/lib/openqa/share/tests ]] ; then
+  pushd /var/lib/openqa/share/tests
+    for dir in $(ls) ; do
+      pushd "$dir"
+      test -x install_deps.sh && ./install_deps.sh
+      popd
+    done
+  popd
+fi
```

Comment:
The code is bad for multiple reasons, e.g. it breaks if white spaces are used and produces noisy `pushd`/`popd` output. I'd just use `find`:
```suggestion
find '/hdd/openqa-devel/openqa/share/tests' -maxdepth 2 -type f -executable -name 'install_deps.sh' -exec {} \;
```